src/models.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy import Column, ForeignKey, Integer, String, DateTime
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+
+# Create the base class for declarative models
+Base = declarative_base()
```

Comment:
I would have created class Base and inherit from DeclarativeBase

Reply:
Done.